src/app/components/product-owner/menu/menu.component.html

```diff
@@ -1,10 +1,13 @@
-<button mat-mini-fab color="primary" (click)="onCreateButtonClicked()">
+<a mat-mini-fab color="primary" href="http://online-planning-poker.herokuapp.com/#/room/participant/{{service.roomId}}" target="_blank" matTooltip="Open link to invite">
```

Comment:
It's a good practice to extract some kind of API builder (that will build your strings for the request paths) or extract some shared constants that will define those paths. This way you have only one source of truth and if the endpoint changes, you can easily change it across the whole app

Reply:
very good remark, I will fix it